source/dyaml/scanner.d

```diff
@@ -1407,6 +1407,12 @@ struct Scanner
                         char[2] escapeSequence = ['\\', cast(char)c];
                         reader_.sliceBuilder.write(escapeSequence);
                     }
+                    // YAML 1.2 supports for JSON optional escape of fwd slashes
+                    else if (c == '/')
+                    {
+                        reader_.sliceBuilder.write(c);
+                        reader_.forward();
```

Comment:
what does forward do? in the backslash branch it's called before write

Reply:
It just advance the scanner to the next char. I can swap them if you'd like.